ConnectedVessels5_4.py

```diff
@@ -0,0 +1,9 @@
+def interleave(*parameters):
+  listReturn=[]
+  for i,element in enumerate(parameters):
```

Comment:
Is there a way to simplify the code here? (for example by using list comprehension?)